src/Microsoft.Extensions.Logging.EventSource/project.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "1.0.0-*",
+  "description": "A logger provider that sends data logged through ILogger interfaces to EventSource/EventListener infrastructure",
```

Comment:
Change to:
`EventSource/EventListener logger provider implementation for Microsoft.Extensions.Logging.`

(this matches the pattern we use in the descriptions of the other loggers)